webapp-django/crashstats/crashstats/middleware.py

```diff
@@ -14,11 +14,18 @@ class SetRemoteAddrFromForwardedFor(object):
     anybody can "fake" their IP address. Only use this when you can absolutely
     trust the value of HTTP_X_FORWARDED_FOR.
     """
+    def __init__(self, get_response=None):
```

Comment:
We have some tests that create a middleware and in the test we don't have a `get_response` value that's helpful, so I did what the Django middlewares and tests do and have it default to `None`. That totally breaks `__call__` since it expects `get_response` to be something. It feels a little weird and skimming the code suggests it's an error.

Maybe we could fix all the tests to pass in a `get_response` that returns a basic Django Response object. Then we wouldn't need the default. Would that be better?

Reply:
I think in the end we'll want to fix the tests and the middleware to be shaped like newer middleware anyway, but what's in this PR now is fine as an in-between if we don't want to do all that rewriting right now.

Reply:
That make sense. +1